internal/stage/filebased/filebased_suite_test.go

```diff
@@ -0,0 +1,20 @@
+package filebased_test
+
+import (
+	"os"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/redhatinsights/insights-ingress-go/internal/config"
+	l "github.com/redhatinsights/insights-ingress-go/internal/logger"
+)
+
+func FileBased(t *testing.T) {
+	tempDir := t.TempDir()
+	os.Setenv("INGRESS_STAGE_FILE_SYSTEM_PATH", tempDir)
```

Comment:
Instead of using the env var, can you try setting the value directly in the `cfg` in the tests that require it?  This approach is very "side-effect-y".  Hopefully it is possible to get a reference to config object, change the config option as needed and pass that into the object/module that needs to be tested.  If not, that's something that we need to fix.

Reply:
Yep, updated this to just set the cfg variables.